crates/sui-framework/sources/clock.move

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/// APIs for accessing time from move calls, via the `Clock`: a unique
+/// shared object that is created at 0x6 during genesis.
+module sui::clock {
+    use sui::object::{Self, UID};
+    use sui::transfer;
+
+    friend sui::genesis;
+
+    struct Clock has key {
+        id: UID,
+        timestamp_ms: u64,
```

Comment:
Do you want to document these fields? Or the type itself?

Reply:
Yep, that is probably worthwhile